src/consts/ibcRoutes.ts

```diff
@@ -0,0 +1,54 @@
+import { IbcRoute, IbcToHypRoute, RouteType } from '../features/tokens/routes/types';
+
+// Configs for manually-defined IBC-only routes
+export const ibcRoutes: Array<IbcRoute | IbcToHypRoute> = [
+  {
+    type: RouteType.IbcNativeToIbcNative,
+    baseTokenCaip19Id: 'cosmos:celestia/ibcDenom:cosmos100000000000000000000000000000000000000',
+    originCaip2Id: 'cosmos:celestia',
+    originDecimals: 6,
+    destCaip2Id: 'cosmos:neutron-1',
+    destDecimals: 6,
+    originIbcDenom: 'utia',
+    sourcePort: 'transfer',
+    sourceChannel: 'channel-8',
+    // IBC Tia on Neutron
+    derivedIbcDenom: 'ibc/773B4D0A3CD667B2275D5A4A7A2F0909C0BA0F4059C0B9181E680DDF4965DCC7',
+  },
+  {
+    type: RouteType.IbcNativeToHypNative,
+    baseTokenCaip19Id: 'cosmos:celestia/ibcDenom:cosmos100000000000000000000000000000000000000',
+    originCaip2Id: 'cosmos:celestia',
+    originDecimals: 6,
+    destCaip2Id: 'ethereum:169',
+    destDecimals: 6,
+    originIbcDenom: 'utia',
+    sourcePort: 'transfer',
+    sourceChannel: 'channel-8',
+    // IBC Tia on Neutron
+    derivedIbcDenom: 'ibc/773B4D0A3CD667B2275D5A4A7A2F0909C0BA0F4059C0B9181E680DDF4965DCC7',
+    intermediateCaip2Id: 'cosmos:neutron-1',
+    // Router on Neutron for Manta
+    intermediateRouterAddress: 'neutron1ch7x3xgpnj62weyes8vfada35zff6z59kt2psqhnx9gjnt2ttqdqtva3pa',
+    // Router on Manta
+    destRouterAddress: '0x6Fae4D9935E2fcb11fC79a64e917fb2BF14DaFaa',
+  },
+  {
+    type: RouteType.IbcNativeToHypNative,
```

Comment:
still think this should be derived from cross product of `ibcRoutes` and `hypRoutes` but not important atm
does `computeTokenRoutes` do something similar?

Reply:
Sure but really this entire file shouldn't exist, not just the ToHypNative route. Ideally these routes would be configured in a way similar to the `consts/tokens.ts` file we already have.